internal/providers/terraform/aws/api_gateway_rest_api.go

```diff
@@ -0,0 +1,159 @@
+package aws
+
+import (
+    "github.com/infracost/infracost/internal/schema"
+    "github.com/shopspring/decimal"
+)
+
+func GetApiGatewayRestApiRegistryItem() *schema.RegistryItem {
+    return &schema.RegistryItem{
+        Name:  "aws_api_gateway_rest_api",
+        RFunc: NewApiGatewayRestApi,
+    }
+}
+
+func NewApiGatewayRestApi(d *schema.ResourceData, u *schema.ResourceData) *schema.Resource {
+    region := d.Get("region").String()
+
+    var apiTierRequests = map[string]decimal.Decimal{
+        "tierOne":   decimal.Zero,
+        "tierTwo":   decimal.Zero,
+        "tierThree": decimal.Zero,
+        "tierFour":  decimal.Zero,
+    }
+
+    monthlyRequests := decimal.Zero
+
+    if u != nil && u.Get("monthly_requests.0.value").Exists() {
+        monthlyRequests = decimal.NewFromInt(u.Get("monthly_requests.0.value").Int())
+    }
+
+    apiRequestQuantities := calculateApiRequests(monthlyRequests, apiTierRequests)
+
+    tierOne   := apiRequestQuantities["tierOne"]
+    tierTwo   := apiRequestQuantities["tierTwo"]
+    tierThree := apiRequestQuantities["tierThree"]
+    tierFour  := apiRequestQuantities["tierFour"]
+
+    return &schema.Resource{
+        Name: d.Address,
+        CostComponents: []*schema.CostComponent{
+            {
+                Name:            "Requests (first 333M)",
+                Unit:            "requests",
```

Comment:
Please set `UnitMultiplier: 1000000,` for all of these cost components, that'll result in the output using the new format.